packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.util.ts

```diff
@@ -143,7 +143,15 @@ export const isBrowserMediaErrorName = (errorName: any) => {
  * @param webClientDomain
  * @returns
  */
-export const getBuildType = (webClientDomain): Event['origin']['buildType'] => {
+export const getBuildType = (
+  webClientDomain,
+  markAsTestEvent = false
+): Event['origin']['buildType'] => {
+  // used temporary to test pre join in production without creating noise data, SPARK-468456
+  if (markAsTestEvent) {
```

Comment:
why wouldn't this just be part of the || statement down below as the first condition, which would automatically short circuit and then we wouldn't have 2 blocks of ifs that return 'test'